nwc-frontend/index.html

```diff
@@ -2,18 +2,10 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="${{BASE_PATH}}/vasp.svg" />
+    <link rel="icon" type="image/svg+xml" href="${{VASP_LOGO_URL}}" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>${{VASP_NAME}}</title>
-    <script type="text/javascript">
-      window.NWC_CONFIG = {
-        // TODO: Replace with a proper template lib. LIG-6299
-        UMA_VASP_LOGIN_URL: "${{UMA_VASP_LOGIN_URL}}",
-        VASP_NAME: "${{VASP_NAME}}",
-        VASP_LOGO_URL: "${{VASP_LOGO_URL}}",
-        BASE_PATH: "${{BASE_PATH}}",
-      };
-    </script>
+    <script data-src="${{CONFIG_PATH}}"></script>
```

Comment:
Note: I had to use data-src and replace the whole thing because vite wanted to bundle the js otherwise (which it can't do).

Reply:
I didn't even know that was a thing. The web is gross.

Reply:
Oh, I think that's not a browser thing, that's a framework thing. Must be that the framework is parsing the script tag and setting the `src` to load the JS after loading. Probably not great for perf, since the browser can't prefetch the script, but 🤷